docs/source/guide/dataset_create.md

```diff
@@ -20,26 +20,217 @@ hide_sidebar: true
+
+3. Scroll down to **Access keys** and click **Create access key**.
+4. Select **Other** and note the recommendations provided by AWS. Click **Next**.
+5. Optionally, add a description for the key.
+6. Click **Create access key**.
+7. Copy the access key ID and your secret access key and keep them somewhere safe, or export the key to a CSV file.
+
+    ![Screenshot of the copy icon next to the key](/images/data_discovery/aws_key2.png)
+
+    <div class="admonition attention"><p class="admonition-title">Important</p><p>This is the only time you will be able to copy the secret access key. Once you click <strong>Done</strong>, you will not be able to view or copy it again.</p></div>
+
+8. Click **Done**. 
+   
+{% enddetails %}
+
+### Create a dataset from an AWS S3 bucket
+
+1. From Label Studio, navigate to the Datasets page and click **Create Dataset**. 
+
+    ![Create a dataset action](/images/data_discovery/dataset_create.png)
+
+2. Complete the following fields and then click **Next**:
+
+    <div class="noheader rowheader">
+
+    | | |
+    | --- | --- |
+    | Name | Enter a name for the dataset. |
+    | Description | Enter a brief description for the dataset.  |
+    | Source | Select AWS S3. |
+
+    </div>
+
+3. Complete the following fields: 
+
+    <div class="noheader rowheader">
+
+    | | |
+    | --- | --- |
+    | Bucket Name | Enter the name of the AWS S3 bucket. |
+    | Bucket Prefix | Enter the folder name within the bucket that you would like to use.  For example, `data-set-1` or `data-set-1/subfolder-2`.  |
+    | File Name Filter | Use glob format to filter which file types to sync. For example, to sync all JPG files, enter `*jpg`. To sync all JPG and PNG files, enter `**/**+(jpg\|png)`.<br><br>At this time, we support the following file types: .jpg, .jpeg, .png, .txt, .text |
+    | Region Name | By default, the region is `us-east-1`. If your bucket is located in a different region, enter it here.  |
+    | S3 Endpoint | Enter an S3 endpoint if you want to override the URL created by S3 to access your bucket. |
+    | Access Key ID | Enter the ID for the access key you created in AWS. Ensure this access key has read permissions for the S3 bucket you are targeting (see "Create an AWS access key" above). |
+    | Secret Access Key | Enter the secret portion of the access key you created earlier. |
+    | Session Token | If you are using a session token as part of your authorization (for example, MFA), enter it here. |
+    | Treat every bucket object as a source file | **Enabled** - Each bucket object will be imported as a separate piece of data (unstructured data). This creates a URL for each bucket object.<br>You should leave this option enabled if your bucket contains BLOB storage files such as JPG, MP3, or similar file types. <br><br>**Disabled** - Disable this option if you are using structured data, such as JSON or CSV files.<br><br>**NOTE:** At this time, we only support unstructured data. |
```

Comment:
Ok, just want to massage this a bit, I'm thinking we could word this as something like:
```md
**Enabled** - Each object in the bucket will be imported as a separate record in the dataset.<br>You should leave this option enabled if your data is simply a bucket of unstructured data files such as JPG, MP3, or TXT.
```
I think the note about creating a URL is a technical detail that isn't relevant, but correct me if I'm missing something there.
Also our use of unstructured/BLOB has been pretty loose, but I think this is a more accurate wording (obviously subjective though, and I trust you if you think I'm making it less clear).
Lastly, I want to point out text files explicitly here. The "or similar file types" seems redundant with "files such as" but I'll trust your opinion here as well if you like it there 🙂

Reply:
To square the "disabled" with my phrasing of "enabled", I'm thinking something like:
```md
**Disabled** - Disable this option if you are importing structured data, such as JSON or CSV files.
```

Reply:
Lastly, because it's so painful for us to have this option without the user being able to use it, I just want to call out that we're working on structured data 😅 
I'm thinking something like:
```md
**NOTE:** At this time, we only support importing a bucket of unstructured data, but structured data is coming soon.
```
Granted this is a bit clumsy, but it's honest and the team is actively working on this support